hugegraph-dist/src/assembly/static/bin/init-store.sh

```diff
@@ -30,7 +30,7 @@ cd $TOP
 
 echo "Initializing HugeGraph Store..."
 
-$JAVA -cp $LIB/hugegraph-dist-*.jar -Djava.ext.dirs=$LIB:$PLUGINS \
+$JAVA -cp $LIB/hugegraph-dist-*.jar -Djava.ext.dirs=$LIB:$PLUGINS:$JAVA_HOME/jre/lib/ext \
```

Comment:
add $EXT var for "$JAVA_HOME/jre/lib/ext" and set -Djava.ext.dirs=$EXT:$LIB:$PLUGINS